internal/provider/resource_azure_cloud_router_connection.go

```diff
@@ -296,6 +306,9 @@ func extractAzureExpressRouteConn(d *schema.ResourceData) packetfabric.AzureExpr
 	if speed, ok := d.GetOk("speed"); ok {
 		expressRoute.Speed = speed.(string)
 	}
+	if subscriptionTerm, ok := d.GetOk("subscription_term"); ok {
+		expressRoute.SubscriptionTerm = subscriptionTerm.(int)
```

Comment:
Nothing for this PR, but the naming in this whole file is off - expressRoute is something entirely different than an Azure Cloud Router Connection.

Reply:
Yeah... there's definitely some of that throughout the provider.